adafruit_stmpe610.py

```diff
@@ -291,13 +300,13 @@ class Adafruit_STMPE610_I2C(Adafruit_STMPE610):
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        i2c,
-        address=_STMPE_ADDR,
-        calibration=None,
-        size=None,
-        disp_rotation=0,
-        touch_flip=(False, False),
-    ):
+        i2c: I2C,
+        address: int = _STMPE_ADDR,
+        calibration: Optional[Tuple[int, int]] = None,
```

Comment:
Looks like `calibration` is a pair of nested tuple:  `Optional[Tuple[Tuple[int, int], Tuple[int, int]]]`.